shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -372,6 +372,10 @@ @implementation FlutterViewController {
   std::shared_ptr<flutter::AccessibilityBridgeMac> _bridge;
 
   FlutterViewId _id;
+
+  // FlutterViewController does not actually uses the synchronizer, but only
+  // passes it to FlutterView.
+  __weak FlutterThreadSynchronizer* _weakViewThreadSynchronizer;
```

Comment:
I don't think this needs to be weak.

Reply:
Changed to strong, especially since `FVC` now directly uses it.